### PR TITLE
update to devs in pom.xml + directory exclusion in gitignore

### PR DESCRIPTION
**Pull request by PaulLodge**

### CODE REVIEW

**Review by AdamKrajcik — APPROVED** [review]
LGTM.

File: pom.xml [hunk 1]
```diff
@@ -28,13 +28,18 @@
...
+        <developer>
+            <id>akrajcik</id>
+            <name>Adam Krajcik</name>
+            <organization>Red Hat</organization>
+        </developer>
       <developer>
-          <id>karm</id>
-          <name>Michal Karm Babacek</name>
+          <id>sgala</id>
+          <name>Sanitago Gala</name>
```
Comment (AdamKrajcik) [inline pom.xml:42]: Typo.

**Review by AdamKrajcik — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (PaulLodge) [reply pom.xml:42]: oops yep 🤪

**Review by AdamKrajcik — APPROVED** [review]
LGTM.

**Review by sgala — APPROVED** [review]
Everything looks right to me

**Pull request merged**